From 3d1b856d5d57d959a06cd15d887e793ad164178b Mon Sep 17 00:00:00 2001
From: Hien Huynh <hien.huynh.px@renesas.com>
Date: Wed, 16 Sep 2020 10:39:47 +0700
Subject: [PATCH 313/441] arm64: dts: renesas: r8a774e1-hihope-rzg2h: Revert
 disabling mode HS400 for r8a774e1

This commit reverts to support mode HS400 for eMMC on Hihope RZ/G2H.

Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
Signed-off-by: Hien Huynh <hien.huynh.px@renesas.com>
---
 arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts b/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
index b398bec..50b1da0 100644
--- a/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
+++ b/arch/arm64/boot/dts/renesas/r8a774e1-hihope-rzg2h.dts
@@ -98,10 +98,6 @@
 		"dclkin.0", "dclkin.1", "dclkin.3";
 };
 
-&sdhi3 {
-	/delete-property/ mmc-hs400-1_8v;
-};
-
 &xhci0 {
 	memory-region = <&global_cma>;
 };
-- 
2.7.4

